src/components/pages/auth-content.tsx

```diff
@@ -10,10 +11,11 @@ type AuthContentProps = {
 const AuthContent: FC<AuthContentProps> = ({ children, className }) => {
   return (
     <div className={cn('h-screen w-screen bg-gradient-blue-white', className)}>
-      <Content className="flex items-center justify-center dark:bg-none dark:bg-base-gray-8">
-        <section className="p-7 sm:p-10 flex gap-7 md:gap-[3.75rem] rounded-3xl bg-base-white dark:bg-base-black">
-          {children}
-        </section>
+      <Content className="flex items-start justify-center dark:bg-none dark:bg-base-gray-8 md:items-center">
```

Comment:
Fix background color according to the design

```suggestion
      <Content className="flex items-start justify-center dark:bg-base-gray-9 md:items-center">
```

Reply:
Hehe, I'm a genius, aren't I?